swift-ui-base/Common/View/TextField/Model/TextFieldData.swift

```diff
@@ -11,16 +11,16 @@ import Foundation
 final class TextFieldData: ObservableObject {
   var value: String = "" {
     didSet {
-      hasTyped = hasTyped || !value.isEmpty
       validate()
     }
   }
+  
   var validationType: ValidationType
   var errorMessage: String
   var title: String
   var isSecure = false
-  var hasTyped = false
-  var isValid = true
+
+  var isValid = true//@Published var isValid = true
```

Comment:
Do you need the commented var?

Reply:
Need to remove that, was there because I was testing other stuff